controllers/k8ssandra/reaper.go

```diff
@@ -267,6 +275,22 @@ func getSingleReaperDcName(kc *api.K8ssandraCluster) string {
 	return ""
 }
 
+func usesHttpAuth(kc *api.K8ssandraCluster, actualDc *cassdcapi.CassandraDatacenter) (bool, *cassdcapi.ManagementApiAuthManualConfig) {
```

Comment:
Suggestion: this looks like a getter with some defaulting behaviour. Can I suggest that it should be a method on the `ManagementApiAuth` type.

Reply:
Understood, apologies I didn't realise it was in cass-operator.